projects/workflows-creator/bundle-element.ts

```diff
@@ -0,0 +1,44 @@
+import * as crypto from 'crypto';
+import {ensureDir, readFileSync, writeFileSync} from 'fs-extra';
+const {copyFile, rm} = require('fs-extra');
+const concat = require('concat');
+async function elementsBundler() {
+  const space = 4;
+  const files = [
+    '../../dist/workflows-element/runtime.js',
+    '../../dist/workflows-element/polyfills.js',
+    '../../dist/workflows-element/main.js',
+  ];
+  await ensureDir('../../dist/workflows-element/dist');
```

Comment:
dist inside dist ?
make this a part of projects/workflow-ele/dist

Reply:
ok sure i m doing according to Search-element "dist/workflow-element/element"